diagnostic.md

```diff
@@ -33,7 +43,9 @@ y = 10
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
+x === 3
+y === 10
+The values of the variables change, The variables themselves do not.
```

Comment:
Hey you didn't include the initial values of `x` and `y`, they start out as 4 and 3 respectively, then they are changed to the ones that you listed. I think you knew that though so good job.